### ISSUE TITLE
neewb: can one use meta edit to quick add tags?

### ISSUE THREAD

**Issue by zeltak** [chhoumann/MetaEdit#34]
Hi!

im not super technical (an academic) and very new to obsidian in general

after exploring obsidian for a few days i moved all my notes there and added my tags in proper YAML format ie:

```
---
tags: [ american,  bread]
priority: TODO
rating: 0
---
```

can one one use meta edit to quick add tags? ie quickly add used tags like `American` without having to type the whole word?
or am i missing the purpose of the plugin completely :) 

thx so much !

Z

**Comment by chhoumann** [comment]
Yes - and it works a lot better now than when you asked. Two ways:

**Quick add with autocomplete** - run **MetaEdit: Run** (give it a hotkey, or right-click → **Edit Meta**) → pick **tags** → **Add to end** → start typing and pick from your existing tags. It's saved as a proper YAML list, and suggestions are drawn from tags across your vault (most-used first).

**Tick-box picker** - if you reuse the same tags, set up an Auto Property: Settings → MetaEdit → **Auto Properties** → add `tags`, set type **Multi**, list your common tags. Editing `tags` then gives you a checklist instead of a text box.

Obsidian's built-in Properties editor also autocompletes tags now, if you just want to add one quickly. Shout if you hit a snag.
